.gitignore

```diff
@@ -10,3 +10,7 @@ project.xcworkspace/
 /.swiftpm
 /Vendor/antlr4-graphql-grammar
 /antlr4.jar
+GraphQL.interp
+GraphQL.tokens
+GraphQLLexer.interp
+GraphQLLexer.tokens
```

Comment:
These are repository wide ignores, prefer to make it path specific.
```
/Sources/GraphQLLanguage/Generated/*.interp
/Sources/GraphQLLanguage/Generated/*.tokens
```

Reply:
Sounds good, will update